airflow/dags/fda_excluded/dag.py

```diff
@@ -5,26 +5,30 @@
 
 from common_dag_tasks import  extract, transform, get_ordered_sql_tasks, get_ds_folder
 from sagerx import read_sql_file
+from airflow.decorators import dag,task
 from airflow.providers.postgres.operators.postgres import PostgresOperator
 
 
 dag_id = "fda_excluded"
 
 dag = create_dag(
     dag_id=dag_id,
-    schedule= "0 4 * * *",  # run at 4am every day
-    start_date=pendulum.yesterday(),
```

Comment:
Can you explain the start date removal?

Reply:
This is another change that I don't think I made - I had no reason to touch any of the FDA Dags as they were working correctly throughout

Reply:
Please remove changes that you did not make or think they should be part of the PR. You can do this by rebasing to master and restoring files. Or by editing them back manually.

Reply:
ok, should be done here I think. I left the time at 4:30 since that seemed to be done in a few other places as well